lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -918,8 +916,13 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
                 variant_url_value = 'cisco-wrlinux'
                 version = '7'
               end
-              remote = "%s/puppetlabs-release%s-%s-%s.noarch.rpm" %
-                [opts[:release_yum_repo_url], repo_name, variant_url_value, version]
+              if repo_name == 'pc1'
+                remote = "%s/puppetlabs-release-%s-%s-%s.noarch.rpm" %
+                  [opts[:release_yum_repo_url], repo_name, variant_url_value, version]
+              else
+                remote = "%s/puppet%s/puppet%s-release-%s-%s.noarch.rpm" %
+                  [opts[:release_yum_repo_url], repo_name, repo_name, variant_url_value, version]
+              end
```

Comment:
I tried to keep this as backward compatible as possible, assuming that folks are no longer using the old puppetlabs repos that came before PC1. If people are still using those pre-pc1 repos, they are way way out of date